src/screens/About/TeamTable/TeamTable.style.tsx

```diff
@@ -1,20 +1,33 @@
 import styled from 'styled-components';
 import Typography from '@material-ui/core/Typography';
 
-export const TeamTableGrid = styled.div``;
-export const TeamMember = styled.div`
+export const TeamTableGrid = styled.div<{
+  isTeam?: Boolean;
+}>`
+  column-count: 1;
+  display: inline-block;
+
+  @media (min-width: 600px) {
+    column-count: 2;
+  }
+`;
+
+export const TeamMember = styled.div<{
+  isTeam?: Boolean;
+  isAdditionalAdvisors?: Boolean;
```

Comment:
Are `isTeam` and `isAdditionalAdvisors` used?  If not, please remove them from the TeamTable.tsx code that passes them in too?